src/app/_components/SearchInput.tsx

```diff
@@ -24,15 +22,24 @@ export function SearchInput({
         <input
           id={id}
           name="search"
-          className="form-input peer block w-full rounded-lg border border-brand-300 bg-brand-800 py-3 pl-10 pr-3 focus:brand-outline placeholder:text-brand-300 hover:border-brand-400 placeholder:hover:text-brand-400 focus:text-brand-100 placeholder:focus:text-brand-100 "
+          className="peer form-input block w-full rounded-lg border border-brand-300 bg-brand-800 py-3 pl-10 pr-9 focus:brand-outline placeholder:text-brand-300 hover:border-brand-400 placeholder:hover:text-brand-400 focus:text-brand-100 placeholder:focus:text-brand-100 [&::-webkit-search-cancel-button]:appearance-none"
           placeholder="Search"
           type="search"
           value={searchQuery}
-          onChange={onSearchChange}
+          onChange={(event) => onSearchChange(event.target.value)}
         />
-        <div className="pointer-events-none absolute inset-y-0 left-0 flex items-center pl-3 text-brand-300 peer-hover:text-brand-400 peer-focus:text-brand-100">
+        <div className="peer pointer-events-none absolute inset-y-0 left-0 flex items-center pl-3 text-brand-300 peer-hover:text-brand-400 peer-focus:text-brand-100">
           <Icon component={MagnifyingGlass} />
         </div>
+        {searchQuery && (
+          <button
+            className="absolute inset-y-0.5 right-0.5 flex items-center rounded-md px-3 text-brand-300 focus:brand-outline peer-hover:text-brand-400 peer-focus:text-brand-100"
+            aria-label="Clear search input"
+            onClick={() => onSearchChange('')}
+          >
+            <Icon component={X} size={16} weight="bold" />
+          </button>
+        )}
       </div>
     </div>
```

Comment:
Would you mind walking @filipagr through the implementation in a screen share - and show her the different states? My only comment is that we might want the focus state to look a bit more square like the menu button. 🙏🏼 Pending Filipa's approval, good to go.

Reply:
Makes sense! I'm gonna push a CR to make it square. Thank you!

Reply:
@filipagr's quiet approval 

![CleanShot 2024-07-19 at 15 33 37](https://github.com/user-attachments/assets/b86d5719-0962-45f3-a49b-e01f777c4e91)